cg_lims/EPPs/udf/calculate/adjust_missing_reads.py

```diff
@@ -0,0 +1,239 @@
+import logging
+import sys
+
+import click
+from cg_lims import options
+from cg_lims.exceptions import LimsError
+from cg_lims.get.artifacts import get_artifacts
+from genologics.entities import Artifact
+
+LOG = logging.getLogger(__name__)
+
+
+def calculate_adjusted_reads(artifact: Artifact, factor: str) -> float:
+    """A function to calculate the adjusted reads to sequence for each artifact with the desired apptag"""
+
+    reads = artifact.udf.get("Reads to sequence (M)")
+    return round(float(reads) * float(factor), 1)
+
+
+def adjust_wgs_topups(
+    artifact: Artifact, factor_wgs_lower: str, factor_wgs_higher: str, threshold_reads: str
+) -> None:
+    """A function that calculates adjusted reads to sequence for WGS topups, where the 'topup' factor is determined
+    by a threshold for the reads to sequence. This is specified in the cli"""
+
+    valid_value = validate_udf_values(artifact=artifact)
+
+    if valid_value:
+        reads = float(artifact.udf.get("Reads to sequence (M)"))
+        if reads < float(threshold_reads):
+            adjusted_reads = round(float(reads) * float(factor_wgs_lower), 1)
+        else:
+            adjusted_reads = round(float(reads) * float(factor_wgs_higher), 1)
+        artifact.udf["Reads to sequence (M)"] = str(adjusted_reads)
+        artifact.put()
+
+
+def reset_microbial_reads(artifact: Artifact, reset_microbial_reads: str) -> None:
```

Comment:
```suggestion
def reset_microbial_reads(artifact: Artifact, reset_microbial_reads: str) -> None:
```
Could we rename the reset_microbial_reads variable so it doesn't share name with the function? Maybe reset_reads, or new_reads_amount. etc